src/styles/brackets_scrollbars.less

```diff
@@ -44,7 +44,7 @@
 .dark.platform-linux .quiet-scrollbars {
 
     ::-webkit-scrollbar {
-        width: 9px;
+        width: 0px;
```

Comment:
Hey can you please use just ``0`` for this? The unit is not needed on ``0`` values.